devel/create_distro_tarball.sh

```diff
@@ -65,163 +66,130 @@ src/build/gyp_chromium -D use_system_libs=1 \$*
 SCRIPT_END
 }
 
-if [ $# -lt 2 ]; then
+if [ $# -lt 1 ]; then
   usage
   exit
 fi
 
-CHECKOUT_DIR=
-if [ "$1" = "--checkout_dir" ]; then
-  if [ -z $2 ]; then
-    usage
-  fi
-  CHECKOUT_DIR=$2
-  shift 2
-else
-  CHECKOUT_DIR=$(mktemp -d)
-  echo $CHECKOUT_DIR
-fi
-
 MINIMAL=0
 if [ "$1" == "--minimal" ]; then
   MINIMAL=1
   shift 1
 fi
 
-TARBALL=$1
-if [ -z $TARBALL ]; then
+TARBALL="$1"
+if [ -z "$TARBALL" ]; then
   usage
 fi
-touch $TARBALL
-TARBALL=$(realpath $TARBALL)
+touch "$TARBALL"
+TARBALL="$(realpath $TARBALL)"
+MPS_CHECKOUT="$PWD"
 
-if [ "$2" == --checkout ]; then
-  if [ -z $3 ]; then
-    usage
-  else
-    GIT_TAG=$3
-    if [ -z $CHECKOUT_DIR ]; then
-      CHECKOUT_DIR=$(mktemp -d)
-    fi
-    SRC_DIR=$CHECKOUT_DIR
-    cd $CHECKOUT_DIR
-    git clone https://github.com/pagespeed/mod_pagespeed.git src
-    cd src
-    git checkout "$GIT_TAG"
-    git submodule update --init --recursive
-  fi
-else
-  SRC_DIR=$2
-fi
+git submodule update --init --recursive
```

Comment:
We need this because otherwise if someone runs this in a clean client it will fail to find all the deps.

(I've tested this by getting a raw Ubuntu 14 LTS VM and running "git clone mps" and then running this script, which now succeeds.  In addition to verifying that it succeeds on our workstations.)